RELEASE_PROCESS.md

```diff
@@ -7,6 +7,7 @@ This document is aimed at members of the Pony team who might be cutting a releas
 In order to do a release, you absolutely must have:
 
 * Commit access to the `ponyc` repo
+* The [changelog tool](https://github.com/ponylang/changelog-tool) installed
```

Comment:
o dur, that was part of the .travis_script.sh

so its installed on travis to verify that the entries are correct, yes?